### ISSUE TITLE
Can this package be run on a Jetson Xavier NX or a Jetson Orin NX (boards with Jetpack version 4.6.1 SDK or later)?

### ISSUE THREAD

**Issue by AdityaVaradaraj** [yanqi1811/stereoDNN#1]
Can this package be run on a Jetson Xavier NX or a Jetson Orin NX (boards with Jetpack version 4.6.1 SDK or later)? Can instructions for the same be provided? @Alexey-Kamenev @vijay609 @nsmoly @yanqi1811

**Comment by nsmoly** [comment]
Yes, it should run on Jetson Xavier NX and Orin with the same instructions.
